cli/utils.go

```diff
@@ -131,3 +132,13 @@ func genericGetCPUDetails() (vendor, model string, err error) {
 
 	return vendor, model, nil
 }
+
+// from runC
+// parseBoolOrAuto returns (nil, nil) if s is empty or "auto"
+func parseBoolOrAuto(s string) (*bool, error) {
```

Comment:
I know this is a small function, but I think its better to include the runc license in the file if not included since we have taken this code varbatim. cc @grahamwhaley wdyt?

Reply:
fixed, thanks